asllib/Parser.mly

```diff
@@ -184,7 +184,7 @@ let nlist(x) :=
   | ~=x; { [ x ] }
   | ~=x; l=nlist(x); { x :: l }
 
-let end_semicolon :=
+let end_semicolon ==
```

Comment:
I wonder why we're not mirroring this non-terminal in the documentation.

Reply:
I added it explicitly to handle the backwards compatibility issue of an optional semicolon easier.

I didn't think it necessary to document it as it should really just be a `terminated_by(END; SEMI_COLON; ..)` once that flag is no longer relevant.